command/destroy_koin.go

```diff
@@ -0,0 +1,72 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hex337/alex-koin-go/model"
+
+	"log"
+	"regexp"
+)
+
+type DestroyKoinCommand struct{}
+
+func (c *DestroyKoinCommand) Run(msg string, event *CoinEvent) (string, error) {
+	re := regexp.MustCompile(`^(?i)destroy koin (?:for )?\<@(?P<to_slack_id>[0-9A-Z]+)\> (?:for)??(?P<reason>.+)`)
+	matches := re.FindStringSubmatch(event.Message)
+
+	if len(matches) < 3 {
+		return "Invalid create koin format. Expected something like `@Alex Koin destroy koin @alexk for being amazing`. See the channel details for command syntax.", nil
+	}
+
+	toUserId := matches[1]
+	reason := matches[2]
+
+	// Get or create the to user
+	toUser, err := model.GetOrCreateUserBySlackID(toUserId)
+
+	if err != nil {
+		log.Fatalf("Could not find user with slack id %s: %s", toUserId, err.Error())
+		return "", err
+	}
+
+	canCreate, msg := canDestroyCoin(event.User, toUser)
+
+	if !canCreate {
+		return msg, nil
+	}
+
+	koin, err := toUser.GetCoin()
+
+	if err != nil {
+		log.Fatalf("Failed to Destroy Koin. Err: %s", err)
+		return fmt.Sprintf("There was a failure in the system. Koin not destroyed"), nil
+	}
+
+	err = koin.DestroyKoin()
```

Comment:
```suggestion
	err = coin.DestroyKoin()
```

Reply:
yo, plz explain to me why our command is `create koin` but we call it `coin` in the app :|

Reply:
koin is a localization.

Reply:
for slack? Can we please add support for this to be more flexible XD like https://github.com/ruby-i18n/i18n